documentation/course/e2/README.md

```diff
@@ -59,7 +59,7 @@ Wyróżnione zostały wśród wymagań z etapu 1 następujące cele, mające wp
   2. dotyczące kont w czasie poniżej 2 sekundy dla co najmniej 90% przypadków.
   3. dotyczące płatności w czasie poniżej 10 sekundy dla co najmniej 90% przypadków.
   4. dotyczące logistyki w czasie poniżej 1 sekundy dla co najmniej 90% przypadków.
-- `NF/PRF/02` - System powinien działać bez zarzutu przy jednoczesnym korzystaniu przez 5000 użytkowników (zgodnie z danymi MPK Wrocław, dziennie korzysta z komunikacji miejskiej pół miliona pasażerów, co przy średnim szacowanym czasie korzystania z aplikacji wynoszącym 3 minuty daje średnio około 1000 użytkowników aplikacji w danym momencie).
+- `NF/PRF/02` - System powinien działać bez zarzutu przy jednoczesnym korzystaniu przez 5000 użytkowników.
```

Comment:
I think that we shouldn't repeat ourselves

Reply:
I mean, fair, but this stage is not about defining those requirements, so why do we have to explain rationale behind them? Whats are your opinions? @tchojnacki @piterek130 

I'll link to `e1/README.md`. We still have some time, we can remove it later anyway.

Reply:
I propose deleting the reasoning. It was a footnote in E1 (I hope it still is) so I would say that the calculations aren't a part of the requirement name.